Apps/phslashnextphishingincidentresponse/slashnextphishingincidentresponse.json

```diff
@@ -0,0 +1,2255 @@
+    "main_module": "slashnextphishingincidentresponse_connector.pyc",
+    "min_phantom_version": "4.5.15922",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "api_key": {
+            "description": "Your API Key (The system uses this API key to authenticate with SlashNext Cloud. If you don\u2019t have a valid API key, please reach us at support@slashnext.com)",
+            "data_type": "password",
+            "required": true,
+            "order": 0
+        },
+        "api_base_url": {
+            "description": "SlashNext API Base URL (Input only if specifically provided by SlashNext)",
+            "data_type": "string",
+            "required": false,
+            "default": "https://oti.slashnext.cloud/api",
+            "order": 1
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity and authentication using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "api quota",
+            "identifier": "api_quota",
+            "description": "Find information about your API quota, like current usage, quota left, etc",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ],
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "width": 6,
+                "height": 10,
+                "view": "slashnextphishingincidentresponse_view.display_api_quota",
+                "title": "SlashNext Phishing Incident Response - API Quota",
```

Comment:
This looks like the encoded `-`. Not sure if this will alter functionality of app/platform, but please verify